bin/tictactoe

```diff
@@ -1,3 +1,32 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+play_again = ""
+until play_again == "stop" || play_again =="Stop"
+
+  puts "Welcome to Tic Tac Toe!\n How many players today?"
+  puts "2"
+  puts "1"
+  puts "0"
+  puts "Player 1 will be X Player 2 will be O"
+  num_player = gets.strip
+
+  until num_player =="2" || num_player == "1" || num_player == "0"
+    num_player = gets.strip
+  end
+
+  if num_player == "2"
+    game = Game.new
+  elsif num_player == "1"
+    game = Game.new(Player::Human.new("X"), Player::Computer.new("O"))
+  else num_player == "0"
+    puts "MACHINE MADNESS!"
+    game = Game.new(Player::Computer.new("X"), Player::Computer.new("O"))
+  end
+
+  game.play
+
+  puts "Would you like to play again?"
+  puts "Stop or one more game"
+play_again = gets.strip
+end
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.